prompting/miners/hf_miner.py

```diff
@@ -50,28 +49,42 @@ def __init__(self, config=None):
         super().__init__(config=config)
 
         model_kwargs = None
-        if self.config.neuron.load_quantized:
-            bt.logging.info("Loading quantized model...")
+        if self.config.neuron.load_in_8bit:
+            bt.logging.info("Loading 8 bit quantized model...")
             model_kwargs = dict(
                 torch_dtype=torch.float16,
                 load_in_8bit=True,
             )
 
+        if self.config.neuron.load_in_4bit:
+            bt.logging.info("Loading 4 bit quantized model...")
+            model_kwargs = dict(
+                torch_dtype=torch.float32,
+                load_in_4bit=True,
+            )
+
         if self.config.wandb.on:
-            self.identity_tags = ("zephyr_miner",)
+            self.identity_tags = ("hf_miner",)
+
+            if self.config.neuron.load_in_8bit:
+                self.identity_tags += ("8bit_quantization",)
+            elif self.config.neuron.load_in_4bit:
+                self.identity_tags += ("4bit_quantization",)
 
-            if self.config.neuron.load_quantized:
-                self.identity_tags += ("8bits_quantization",)
+        # Forces model loading behaviour over mock flag
+        mock = (
+            False if self.config.neuron.should_force_model_loading else self.config.mock
```

Comment:
?

Reply:
This logic is there to facilitate the use of a miner running everything mocked but the model, useful for benchmarks